src/ManifoldMuseum.jl

```diff
@@ -278,8 +278,8 @@ injectivity_radius(M::Manifold, x) = 1.0
 """
     injectivity_radius(M::Manifold, x, R::AbstractInverseRetractionMethod)
 
-Distance such that `retract(M, x, y, R)` is defined for all points within
-this radius.
+Distance such that `inverse_retract(M, x, y, R)` is injective for all points
+within this radius (has a left inverse).
```

Comment:
I think it's the other way around. For example, `exp(M, x, log(M, x, y))` always produces `y`, but `log(M, x, exp(M, x, v))` does not always equal `v`. So it's the radius in which `retract` is injective.

Reply:
Makes sense, I'll change the docstring for the other method too.